company/beliefs_and_values.md

```diff
@@ -9,3 +10,37 @@
 * We believe that nearly all organisations can move faster than they do. We try to help them to do this.
 
 * We believe the adoption of Agile is in it's infancy and that Agile principles (and various spin offs) will underpin all high-performance businesses in years to come.
+
+###Values
+
+* We take care of the our space and those in it.
+
+* We are willing and open to continuous improvement in our culture
+
+* We take care of those around us and those we talk to.
+
+* No one should feel uncomfortable
+
+* Everyone should feel productive
+
+* Be aware of ego
+
+* Actions over words
+
+* Don't be submissive
+
+* Know your audience
+
+* Awareness of culture
+
+* Adapt and iterate culture
+
+* Seek diversity
+
+* Be responsible for our space
+
+* Scout attitude to the office
+
+* Prioritise our environment
+
+* Respect others environments
```

Comment:
Tried to distill that list down into a few points:

- We value and respect the people we work with, both in the office and out, and want them to feel able to express their opinions safely and without fear.

- We value the space in which we work, and want it to be a place where everyone feels respected, comfortable and productive.

- We value a positive and diverse company culture, and will always strive to adapt our culture in order to make this a more welcoming place to be.